action.yml

```
@@ -18,6 +18,10 @@ inputs:
     description: "delete the previously created comment. Only `true` is allowed. Just skip this item when you don't need it."
     default: "false"
     required: false
+  only_create:
+    description: "only create a new comment if there is no existing one, otherwise do nothing. Only `true` is allowed. Just skip this item when you don't need it."
```

I get inquiries from time to time, so I want to specify the priority.
```suggestion
    description: "only create a new comment if there is no existing one, otherwise do nothing. Only `true` is allowed. Just skip this item when you don't need it. This options has higher priority then hide_and_recreate, hide"
```